docs/User/ProjectRelated/ApiChangelogs/0.8.3-0.9.0.txt

```diff
@@ -0,0 +1,175 @@
+    [A] parList :: MonadAsync m => (Config -> Config) -> [Stream m a] -> Stream m a
+    [A] parEval :: MonadAsync m => (Config -> Config) -> Stream m a -> Stream m a
+    [A] parConcatMap :: MonadAsync m => (Config -> Config) -> (a -> Stream m b) -> Stream m a -> Stream m b
+    [A] parConcatIterate :: MonadAsync m => (Config -> Config) -> (a -> Stream m a) -> Stream m a -> Stream m a
+    [A] parConcat :: MonadAsync m => (Config -> Config) -> Stream m (Stream m a) -> Stream m a
+    [A] parApply :: MonadAsync m => (Config -> Config) -> Stream m (a -> b) -> Stream m a -> Stream m b
+    [A] ordered :: Bool -> Config -> Config
+    [A] minRate :: Double -> Config -> Config
+    [A] maxThreads :: Int -> Config -> Config
+    [A] maxRate :: Double -> Config -> Config
+    [A] maxBuffer :: Int -> Config -> Config
+    [A] intervalsOf :: MonadAsync m => Double -> Fold m a b -> Stream m a -> Stream m b
+    [A] interleaved :: Bool -> Config -> Config
+    [A] interject :: MonadAsync m => m a -> Double -> Stream m a -> Stream m a
+    [A] inspect :: Bool -> Config -> Config
+    [A] fromCallback :: MonadAsync m => ((a -> m ()) -> m ()) -> Stream m a
+    [A] finally :: (MonadAsync m, MonadCatch m) => m b -> Stream m a -> Stream m a
+    [A] eager :: Bool -> Config -> Config
+    [A] dropInterval :: MonadAsync m => Double -> Stream m a -> Stream m a
+    [A] constRate :: Double -> Config -> Config
+    [A] bracket :: (MonadAsync m, MonadCatch m) => m b -> (b -> m c) -> (b -> Stream m a) -> Stream m a
+    [A] avgRate :: Double -> Config -> Config
+    [A] after :: (MonadIO m, MonadBaseControl IO m) => m b -> Stream m a -> Stream m a
+[A] Streamly.Data.Stream.MkType
+    [A] class Monad m => MonadReader r (m :: Type -> Type) | m -> r
+    [A] class MonadTrans (t :: Type -> Type -> Type -> Type)
+    [A] class Read a
+    [A] class IsString a
+    [A] class IsList l
+    [A] Identity
+    [A] throwM :: (MonadThrow m, Exception e) => e -> m a
+    [A] readsPrec :: Read a => Int -> ReadS a
+    [A] reader :: MonadReader r m => (r -> a) -> m a
+    [A] readPrec :: Read a => ReadPrec a
+    [A] readListPrec :: Read a => ReadPrec [a]
+    [A] readList :: Read a => ReadS [a]
+    [A] mkZipType :: String -> String -> Bool -> Q [Dec]
+    [A] mkCrossType :: String -> String -> Bool -> Q [Dec]
+    [A] local :: MonadReader r m => (r -> r) -> m a -> m a
+    [A] liftIO :: MonadIO m => IO a -> m a
+    [A] lift :: (MonadTrans t, Monad m) => m a -> t m a
+    [A] ask :: MonadReader r m => m r
+    [A] ap :: Monad m => m (a -> b) -> m a -> m b
+[D] Streamly.Data.Fold.Tee
+    [C] Tee
+        [A] [unTee] :: Tee (m :: Type -> Type) a b -> Fold m a b
+        [R] [toFold] :: Tee m a b -> Fold m a b
+        [C] Tee
+            [O] Tee :: Fold m a b -> Tee m a b
+            [N] Tee :: Fold m a b -> Tee (m :: Type -> Type) a b
+    [A] toFold :: forall (m :: Type -> Type) a b. Tee m a b -> Fold m a b
```

Comment:
This is technically correct. We can leave it as it is.
Is is saying `toFold` record functions has been removed and a normal `toFold` functions has been added.

Reply:
I've exposed the `toFold` function just now.